sdk/core/Azure.Core.TestFramework/src/RecordedTestAttribute.cs

```diff
@@ -14,7 +14,7 @@ namespace Azure.Core.TestFramework
 {
     [AttributeUsage(AttributeTargets.Method)]
     /// <summary>
-    /// This attribute replaces the [Test] attribute and will dynamically re-record recorded tests on failure.
+    /// This attribute replaces the [Test] attribute and will dynamically re-record recorded tests on failure when run in Playback mode.
```

Comment:
What happens if this is being run in Playback mode and there aren't test resources deployed to re-record?   I understand they'll fail, but would that potentially cause issues with the session records if someone blindly copies the failed result over?

Reply:
The session will not be saved into the session file if the test failed (unless the [SaveDebugRecordingsOnFailure](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core.TestFramework/src/RecordedTestBase.cs#L108) flag is set).